### ISSUE TITLE
what are the reference papers of this implementation?

### ISSUE THREAD

**Issue by Light--** [docongminh/age-gender-estimator#1]
and what about the performance? What about the accuracy of face recognition and age/gender estimation ? Did you have tested those ?

**Comment by defi0x1** [comment]
HI @Light--  
I did not based on specific paper. Basically, i research many paper about age and gender. After that, I have summarized all papers and combined into this method. Actually, I realized that with this problem data and preprocessing data process important than architecture of network. So i chose simple network architecture.
The accuracy on test set(all images in test set high quality and full face) of gender and age approximate 98,4% and 84%, respectively. And when i tested on real faces. Imagine, I chose 5 or 6 people (male, female, young,  mature people, old people) in front of the camera in 5-6 miu, and got result to evaluate in real-face. In this task gender and age accurate were 83% and 51%. And with other challenge case about Illumination and Pose invariance,... got low accuracy.
